src/app/home/home-gallery/home-gallery.component.html

```diff
@@ -0,0 +1,27 @@
+
+<div style="padding-left: 10%; padding-right:10%">
+  <div class="container">
+    <div class="row" >
+
+        <div class="col-lg-4 col-md-6 col-xs-12 " *ngFor="let links of pictureArray">
```

Comment:
When iterating with ngFor, always use the following format:

`let item of items`

In this particular case, a good practice would be: `let imageLink of imageLinks`